ruby/red-gandiva/lib/gandiva/expression-buildable.rb

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+module Gandiva
+  module ExpressionBuildable
```

Comment:
How about creating `ExpressionBuilder` and just use in `Arrow::Schema#build_expression` instead of putting details to this module?

```ruby
module Arrow
  class Schema
    def build_expression(&block)
      builder = Gandiva::ExpressionBuilder.new(self)
      builder.build(&block)
    end
end 
```

Reply:
I'll do it.
Does that looks ok to add `Arrow::Schema#build_expression` in red-gandiva/lib/gandiva/arrow-schema.rb? Or is it better to add it in red-arrow?

Reply:
We should put it to `red-gandiva/lib/gandiva/arrow-schema.rb` because it doesn't work only with Red Arrow.

Reply:
OK. Thank you for your confirmation.